examples/oracle.yaml

```diff
@@ -0,0 +1,16 @@
+arch: "x86_64"
```

Comment:
Please link `oracle.yaml` in `examples/README.md` too

Reply:
Fixed in the latest push: https://github.com/lima-vm/lima/pull/700/files#diff-49aaa2819e35a856818ecec8c9fa7e1c79ad028d3f44bd749736353cfb51bac9R14